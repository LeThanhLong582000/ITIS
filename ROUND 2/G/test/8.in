46
011101101111011000000111
000101000110111101010001001001100001
1110111000100100011010111000001010111101011010011000010010010100010111001100000
0000100100101010110001
1000001011111000110100
000101100000001100
1110100100101111010110111011011111011101110111111110111
1010010010010011000010101101111101101111001111110100001111011000111111111101010000011001100000100111
001000011001001011001010111011000110100011010100000111001011010110101001011000001011101000100011110
101100001001000101011110000111110110101100100001101000111010100010111110111
01110000001000010110110010010001
00111101100111110000001111001011010
101001010000110111011111111000010110010001010011011110010110001001000000011
0000110010100000011010110011011100110000100011101000010110100100011101000111010101100
101010110001111111101010100001000010010101101001010000
101100111010100011010111111000011
111101101100110110001
100000001010101101110110111001001000101010101101100111000011011100111110110111000111000110
110100100110100100111100001101011001010110010010000100000011100110
1111010110000010000100101001111000101001100010000111011011110101001101010010000111010110
01110101101101000000100100100111100111110000011000101111
0101001010111101011101001011011111111101
010111100100011111000110000000100111111
1000110000110001111011000000010111101011010000001000101000111000101011001111111001001111011
101001011001000001011110011111110110001001110000001
0111110111111101011010111010001110011011110111000111
0000000011111100011000010010011101000011000100100111101111001011100111101
0111000010111101000
110100010110111011110101101001011101111101011000110111100000101100010011001111011010
00001011110001011000110010
11101000111001010100010011001100001101001110111111001101011001100101110100110101110000
1111110000110100111010100001110100110100111111001000001110010010011100110100
00110011000011010110101000110001010011100
11111111101111101111
1011011001110100011000000011111100101001001001000100001000000011000011000000110001010001100011
1111110010010010001001110000000111110100111011110001100111011100100111000001110110
00101001111001100011110010100110001100001100101011110010010101001111111111
1101101100100110110101010000011100100011110101010101010000010110011000000110001100110
011010100010010011111111101100111000001000
1010101100101010000100110011111101001011001110111
01011010001100000000100100101011111111010111111110001100011110111011011001110010111100100000
111000010011010111011011100010000001011010011111011110101010111100110000111111000000
100111100110010111000111001010011
0111001011011101101110010110100100010101011101011
11001010100110011100110110000000111100000010000011001111001101010100110001
11110110000001010110010
