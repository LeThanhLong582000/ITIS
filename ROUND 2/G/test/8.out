7
14
31
7
8
5
15
39
42
31
10
13
29
35
18
12
6
37
27
37
23
12
14
35
20
16
25
7
33
11
37
31
17
2
32
34
30
35
14
18
33
34
14
20
30
6
